ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-bgp.j2

```diff
@@ -495,10 +508,16 @@
 
 | VRF | Route-Distinguisher | Redistribute |
 | --- | ------------------- | ------------ |
-{%         for vrf in router_bgp.vrfs | arista.avd.natural_sort %}
-{%             set route_distinguisher = router_bgp.vrfs[vrf].rd | arista.avd.default("-") %}
-{%             set redistribute = router_bgp.vrfs[vrf].redistribute_routes | arista.avd.default("-") %}
-| {{ vrf }} | {{ route_distinguisher }} | {{ redistribute | join("<br>") }} |
+{%         for vrf in router_bgp.vrfs | arista.avd.convert_dicts('name') | arista.avd.natural_sort('name') %}
+{%             set route_distinguisher = vrf.rd | arista.avd.default("-") %}
+{%             set redistribute_route = vrf.redistribute_routes | arista.avd.default([]) |
+                                                                      arista.avd.convert_dicts('type') %}
+{%             if redistribute_route.type is arista.avd.defined %}
+{%                 set redistribute = redistribute_route.type | arista.avd.default("-") %}
+{%             else %}
+{%                 set redistribute = redistribute_route | arista.avd.default("-") %}
+{%             endif %}
```

Comment:
```suggestion
{%             set redistribute = vrf.redistribute_routes | arista.avd.default([{"type": '-'}]) | arista.avd.convert_dicts('type') | map(attribute='type') %}
```

Reply:
this is not working for the places where type is not there, if u see in yaml files at some places redistributes is having list value and at some places its dictionary